pkgs/top-level/python-packages.nix

```diff
@@ -18440,4 +18440,24 @@ let
     };
   };
 
+  importmagic = buildPythonPackage rec {
+    simpleName = "importmagic";
+    name = "${simpleName}-${version}";
+    version = "0.1.3";
+    doCheck = false;  # missing json file from tarball
```

Comment:
Expliciting this, the tests fail because they are missing a .json file (present in repository but not in tarball).